Error MSB3030: Could not copy the file "X:\...\node_modules\react-native-xaml\Codegen\Windows.UI.Xaml.json"

When attempting to go through the steps to generate the projections for a custom control ([here](https://github.com/microsoft/react-native-xaml/blob/main/TechnicalGuide.md)) I get the following error:

```ps
> dotnet run X:\...\node_modules\react-native-xaml\codegen -- -winmd "X:\path\to\my\component.winmd>"
...
Build FAILED.

       "X:\...\node_modules\react-native-xaml\Codegen\Codegen.csproj" (default target) (1:7) ->
        (_CopyOutOfDateSourceItemsToOutputDirectory target) ->
         C:\Program Files\dotnet\sdk\5.0.401\Microsoft.Common.CurrentVersion.targets(4965,5): error MSB3030: Could not copy
         the file "X:\...\node_modules\react-native-xaml\Codegen\Windows.UI.Xaml.json" because it was not found. 
```

--- a/package/package.json
+++ b/package/package.json
@@ -1,7 +1,7 @@
 {
   "name": "react-native-xaml",
   "title": "React Native Xaml",
-  "version": "0.0.41",
+  "version": "0.0.42",
   "description": "Allows using XAML directly, inside of a React Native Windows app",
   "main": "lib/index.js",
   "typings": "lib/index.d.ts",
@@ -51,6 +51,7 @@
   "files": [
     "!*.tgz",
     "CodeGen/*.cs*",
+    "CodeGen/*.json",
     "windows/",
     "lib/",
     "src/",

